HaPy-haskell/Foreign/HaPy.hs

```diff
@@ -90,7 +90,7 @@ toForeignExp _         exp = fail "conversion failed: unknown type!"
 
 makeFunction :: (String -> String) -> (Name -> [FType] -> ClauseQ) -> ([FType] -> TypeQ) -> Name -> DecsQ
 makeFunction changeName makeClause makeType origName = do
-  VarI _ t _ _ <- reify origName
+  VarI _ t _ <- reify origName
```

Comment:
Is this change backwards-compatible?  If not, we should bump the major version number.

Reply:
Just bumped version number.
Sorry to miss this change before.